src/mito2/src/sst/parquet/writer.rs

```diff
@@ -112,9 +113,11 @@ where
     pub async fn write_all(
         &mut self,
         mut source: Source,
+        override_sequence: Option<SequenceNumber>, // override the `sequence` field from `Source`
         opts: &WriteOptions,
```

Comment:
Can we move the `override_sequence` to `WriteOptions`. It should be an option to control the writer.